internal/observability/ping.go

```diff
@@ -37,18 +43,18 @@ func Ping() {
 		DBConfig:       DBConfig{},
 	}
 
-	url := "https://api.us-east.aws.tinybird.co/v0/events?name=ping"
 	b, _ := json.Marshal(payload)
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+	defer cancel()
 
-	req, _ := http.NewRequestWithContext(context.Background(), "POST", url, bytes.NewBuffer(b))
-	//nolint:lll
-	req.Header.Set("Authorization", "Bearer p.eyJ1IjogIjhjNWQxMjdlLTczZmYtNGRjZS04Mzk5LTQyMDU0MThhYjc2OSIsICJpZCI6ICJhZjcxNGExNC0xZWQyLTQ3ZDktOTM0MS0xMzgwNWNiOWFhNDYiLCAiaG9zdCI6ICJ1cy1lYXN0LWF3cyJ9.o9LqZqTZ9YkhbcusZOltsm95RzVQUzJLQOHV2YA7L0E")
+	req, _ := http.NewRequestWithContext(ctx, "POST", url, bytes.NewBuffer(b))
+	req.Header.Set("Authorization", "Bearer "+token)
 	req.Header.Set("Content-Type", "application/json")
 
 	client := &http.Client{Timeout: time.Second * 5}
 	resp, err := client.Do(req)
 	if err != nil {
 		panic(err)
```

Comment:
@JyotinderSingh please add your thoughts around the `panic` here? Was thinking should we panic if the `PING` fails or we can log error?
We can add integration test around it so that we get to know if token expires or something.

Reply:
As discussed offline, let's not panic here since this is an optional operation.

Reply:
Added error statement, adding sample below:
```
{"level":"error","error":"Post \"https://api.us-east.aws.tinybird.co/v0/events?name=ping\": context deadline exceeded","time":"2024-10-11T22:58:53+05:30","message":"Error reporting observability metrics."}
```